tests/backup/mysql/scheduled_mysql_test.js

```diff
@@ -0,0 +1,99 @@
+const moment = require('moment');
+
+const { locationsPage, psMySql } = inject();
+const connection = psMySql.defaultConnection;
+const location = {
+  name: 'mysql scheduled backup location',
+  description: 'MySQL location for scheduling',
+  ...locationsPage.mongoStorageLocation,
+};
+
+let locationId;
+let serviceId;
+
+const mysqlServiceName = 'mysql-with-backup';
+const schedules = new DataTable(['cronExpression', 'name', 'frequency']);
+
+schedules.add(['30 8 * * *', 'schedule daily', 'At 08:30 AM']);
+schedules.add(['0 0 * * 2', 'schedule weekly', 'At 12:00 AM, only on Tuesday']);
```

Comment:
why do you have this data table if you are not using it?

Reply:
ah yeah! missed to clean up. Fixed